README.md

```diff
@@ -306,6 +306,22 @@ The file containing any policy rule implementations to be used with the _evaluat
 |---------------------|-----------|---------------------------------------|
 | Kotlin script (DSL) | Evaluator | `$ORT_CONFIG_DIR/evaluator.rules.kts` |
 
+### Protecting environment variables
+
+In order to do its analyses, ORT invokes a number of external tools, such as package managers or scanners. Especially when interacting with package managers to obtain the dependencies of the analyzed project, this can lead to the execution of code in build scripts from potentially unknown sources. One possible risk in this constellation is that untrusted code could read sensitive information from environment variables used for the ORT configuration, such as database connection strings or service credentials. This is because the environment variables of a process are by default propagated to the child processes spawned by it.
+
+To reduce this risk, ORT filters out certain environment variables when it runs external tools in child processes. This filter mechanism can be configured via the following properties in the [ORT configuration file](./model/src/main/resources/reference.yml):
+
+| Property | Description |
+|--------|-----------|
+| deniedProcessEnvironmentVariablesSubstrings | A list with substrings that identify variables containing sensitive information. All variables that contain at least one of these strings (ignoring case) are not propagated to child processes. The default for this property contains strings like "PASS", "PWD", or "TOKEN", which are typically used to reference credentials. |
+| allowedProcessEnvironmentVariableNames | This is a list of variable names that are explicitly allowed to be passed to child processes - even if they contain a substring listed in `deniedProcessEnvironmentVariablesSubstrings`. Via this property variables required by external tools, e.g. credentials for repositories needed by package managers, can be passed through, no matter how they are named. Note that variable names must match exactly, and matches are case-sensitive. |
```

Comment:
I would drop the "no matter how they are named" as it reads a bit odd with the following "variable names must match exactly" (although I know what you mean).

Reply:
Change

"Note that variable names must match exactly, and matches are case-sensitive."

to

"Here, entries must match variables names exactly and case-sensitively."